[2022-09-23 14:37:31,982] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: kag_to_gcs_dag.check_list_task manual__2022-09-23T14:31:12.806772+00:00 [queued]>
[2022-09-23 14:37:31,991] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: kag_to_gcs_dag.check_list_task manual__2022-09-23T14:31:12.806772+00:00 [queued]>
[2022-09-23 14:37:31,991] {taskinstance.py:1238} INFO - 
--------------------------------------------------------------------------------
[2022-09-23 14:37:31,991] {taskinstance.py:1239} INFO - Starting attempt 2 of 3
[2022-09-23 14:37:31,991] {taskinstance.py:1240} INFO - 
--------------------------------------------------------------------------------
[2022-09-23 14:37:32,001] {taskinstance.py:1259} INFO - Executing <Task(BashOperator): check_list_task> on 2022-09-23 14:31:12.806772+00:00
[2022-09-23 14:37:32,007] {standard_task_runner.py:52} INFO - Started process 1910 to run task
[2022-09-23 14:37:32,010] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'kag_to_gcs_dag', 'check_list_task', 'manual__2022-09-23T14:31:12.806772+00:00', '--job-id', '9', '--raw', '--subdir', 'DAGS_FOLDER/kag_to_gcs_dag.py', '--cfg-path', '/tmp/tmpjosyvcm1', '--error-file', '/tmp/tmp4go108op']
[2022-09-23 14:37:32,011] {standard_task_runner.py:77} INFO - Job 9: Subtask check_list_task
[2022-09-23 14:37:32,052] {logging_mixin.py:109} INFO - Running <TaskInstance: kag_to_gcs_dag.check_list_task manual__2022-09-23T14:31:12.806772+00:00 [running]> on host 31708a4d5ab1
[2022-09-23 14:37:32,093] {taskinstance.py:1426} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=kag_to_gcs_dag
AIRFLOW_CTX_TASK_ID=check_list_task
AIRFLOW_CTX_EXECUTION_DATE=2022-09-23T14:31:12.806772+00:00
AIRFLOW_CTX_DAG_RUN_ID=manual__2022-09-23T14:31:12.806772+00:00
[2022-09-23 14:37:32,094] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2022-09-23 14:37:32,094] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'ls']
[2022-09-23 14:37:32,109] {subprocess.py:85} INFO - Output:
[2022-09-23 14:37:32,112] {subprocess.py:93} INFO - Command exited with return code 0
[2022-09-23 14:37:32,133] {taskinstance.py:1277} INFO - Marking task as SUCCESS. dag_id=kag_to_gcs_dag, task_id=check_list_task, execution_date=20220923T143112, start_date=20220923T143731, end_date=20220923T143732
[2022-09-23 14:37:32,182] {local_task_job.py:154} INFO - Task exited with return code 0
[2022-09-23 14:37:32,206] {local_task_job.py:264} INFO - 0 downstream tasks scheduled from follow-on schedule check
